build_defs/legacy.build_defs

```diff
@@ -0,0 +1,510 @@
+"""Build rules for compiling protocol buffers & gRPC service stubs.
+Note that these are some of the most complex of our built-in build rules,
+because of their cross-language nature. Each proto_library rule declares a set of
+sub-rules to run protoc & the appropriate java_library, go_library rules etc. Users
+shouldn't worry about those sub-rules and just declare a dependency directly on
+the proto_library rule to get its appropriate outputs.
+It is possible to add extra languages to these for generation. This is accomplished
+via the 'languages' argument; this can be simply a list of languages to build, but
+can also be a mapping of language name -> definition of how to build it. The definition
+should be the return value of proto_language.
+"""
+
+if "go" in CONFIG.PROTO.LANGUAGES:
+    subinclude("///go//build_defs:go")
+if "py" in CONFIG.PROTO.LANGUAGES:
+    subinclude("///python//build_defs:python")
+if "java" in CONFIG.PROTO.LANGUAGES:
+    subinclude("///java//build_defs:java")
+if "cc" in CONFIG.PROTO.LANGUAGES:
+    subinclude("///cc//build-defs:cc")
```

Comment:
I think these should be CONFIG.LANGUAGES. Similarly for other CONFIG.PROTO values

Reply:
That wasn't what I had in mind but maybe you're right. Do you think we should be using the old `[Proto]` section?